Support mil2525c_b2 renderer/style

csmoore opened the pull request:
Changes to the addin to support using the mil2525c_b2 dictionary renderer/style on Pro 1.2.

Primary changes include:
1. Settings form to set the desired standard
2. Supporting 2 different datamodels/styles/lpks
3. Fixes to bugs that arose from 1, 2
4. Updated to latest CoordinateConversionLibrary
5. Changing name from "Military Symbol **Designer**" to "Military Symbol **Editor**" 

~~Opening PR now, but will wait until Addin is further tested to merge~~

kgonzago commented on source/ProSymbolEditor/ViewModels/MilitarySymbolDockpaneViewModel.cs:
```diff
@@ -500,8 +631,18 @@ public SelectedFeature SelectedSelectedFeature
+                    }
+                    catch (Exception exception)
+                    {
+                        System.Diagnostics.Debug.WriteLine(exception.ToString());
```
I know this is for debugging purposes but should be removed. Use another way to capture exception and report to user.

csmoore commented:
@kgonzago - thanks for having a look. Most of those TODOs were just things I noticed when reviewing the existing code (and wanted to note, but minimize changes). That HACK was a data binding/update issue that I was afraid of breaking if I changed - but I do need to dig into that - I added some more comments to better explain that section. Thanks again.
